config/load.go

```diff
@@ -37,12 +36,10 @@ var viperLock sync.Mutex
 
 // Load all environment variables
 func Load(customWorkingDirectory string) (_appConfig *AppConfig, err error) {
-
 	// Check the environment we are running
 	environment := os.Getenv(EnvironmentKey)
 	if !isValidEnvironment(environment) {
 		err = fmt.Errorf(dictionary.GetInternalMessage(dictionary.ErrorInvalidEnv), environment)
-		logger.Data(2, logger.ERROR, err.Error())
```

Comment:
Don't we need logging here ?

Reply:
We do not have logger here and If we will return the error it will be displayed in cmd/main.go